src/gpio.rs

```diff
@@ -239,7 +239,7 @@ pub mod $gpiox {
 
         fn split(self, apb2: &mut APB2) -> Self::Parts {
             riscv::interrupt::free(|_| {
-                apb2.en().write(|w| w.$en().set_bit());
+                apb2.en().modify(|_,w| w.$en().set_bit());
```

Comment:
Why should we perform this change? :) @o080o

Reply:
without this, I was unable to enable more than one gpio port. It seems that write resets all other values, and modify does not.